### PR TITLE
Adding Montreal chapter

### PR DESCRIPTION
**Pull request by pcuci**
Depends on nodeschool/organizers/issues/350

@David-Ro and I will fork the nodeschool/toronto repo to get a jump start

### CODE REVIEW

File: chapters/montreal.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  name: "NodeSchool Montréal",
```
Comment (riyadhalnur) [inline chapters/montreal.json]: Add double quotes to `key` names

**Comment by riyadhalnur** [comment]
Thank you! Triaging in a bit. Do you want to merge this after 350 is closed?

File: chapters/montreal.json (hunk 1 above)
Comment (pcuci) [inline chapters/montreal.json]: Done, pull request updated :-)

**Comment by pcuci** [comment]
@riyadhalnur I'm meeting the Toronto nodeschool folks today, I think we can safely merge - your call, not sure who I need to wait for for 350 to get done.

**Comment by riyadhalnur** [comment]
@pcuci No problem. I merged it in. I think you can close out 350 at your own pace once you have the meeting today.

**Pull request merged**